Tests/KlaviyoSwiftTests/__Snapshots__/EncodableTests/testEventPayload.1.json

```diff
@@ -3,7 +3,7 @@
     "attributes" : {
       "metric" : {
         "name" : "blob",
-        "service" : "ios-analytics"
+        "service" : "klaviyo"
```

Comment:
I changed this back to klaviyo so it shows up properly in the UI. We might think about getting our icon to indicate this is coming from ios but for now this should be left as is.